tests/sentry/api/endpoints/test_user_authenticator_details.py

```diff
@@ -155,6 +163,8 @@ def test_u2f_remove_device(self):
         resp = self.client.delete(url)
         assert resp.status_code == 500
 
+        assert email_log.info.call_count == 0
```

Comment:
Removing a device seems like it should generate an email as well

Reply:
Ya, I'll add it in a follow-up pr ^